cms/config/environments/production.rb

```diff
@@ -86,9 +85,5 @@
       :exception_recipients => %w{tralles@qb-it.de}
```

Comment:
dear @paulmenzel, can you please also alter my email to no-reply@your-domain.tld also here. thanks!